terraform/fec_rds.tf

```diff
@@ -93,13 +93,21 @@ resource "aws_db_instance" "rds_production" {
   publicly_accessible = true
   storage_encrypted = true
   multi_az = true
+  auto_minor_version_upgrade = true
```

Comment:
I see the difference, not sure how I didn't get to what you were looking at, but I somehow got to here: https://www.terraform.io/docs/providers/aws/d/db_instance.html#storage_type

Not as useful. :-)